src/legend.js

```diff
@@ -220,5 +220,14 @@ export default {
       cellsSvg.attr('transform', 'translate(' + xOffset + ',' + (yOffset) + ')');
 
     }
-  }
+  },
+
+  d3_defaultLocaleDefinition: {
+    decimal: '.',
+    thousands: ',',
+    grouping: [3],
+    currency: ['$', '']
+  },
```

Comment:
It looks like in the d3-format module it is published with the locale folder, so for this we can import something like `d3-format/local/en-US.json` and use this as the default definition instead of remaking it here.

Reply:
I agree, this is not the most efficiant way to get the locale. But I'm facing two problems:
1. I cannot access to the default locale, calling `d3.formatLocale()` without args throws an error
2. I cannot import the locales as simple as you said because of https://github.com/d3/d3-format/issues/22 and https://github.com/d3/d3-format/issues/8

Should I import like this?
`require('../node_modules/d3-format/locale/en-US.json')`
I'm not fan

Reply:
I found an another way, a `locale` is no more than an object with a `format` and `formatPrefix` methods, so I created a default one with the package methods.